### ISSUE TITLE
xgettext fails due to differing file names

### ISSUE THREAD

**Issue by Photon89** [udda/multiload-ng#7]
Just a minor thing (the open issue list looked so empty that I went ahead and filled it a bit :) ): I wanted to translate multiload-ng and followed the corresponding howto, however xgettext failed due to files being not found. In particular:

Expected: standalone/multiload-ng.desktop.in
Found: standalone/multiload-ng.desktop.in.in
Expected: xfce4/multiload-ng.desktop.in
Found: xfce4/multiload-ng.desktop.in.in

**Comment by udda** [comment]
Hello, thanks for your report.
I'm actually *happy* to see new open issues, as they show interest in my project and anyway I'm willing to fix them, so thank you for your interest!

I'll look into this in a couple of hours. Looks like a simple naming problem. By the way, which language you are translating into?

(side note: you may want to wait a little bit, because there is an ongoing massive UI rewriting = new translatable strings)

**Comment by Photon89** [comment]
I started translating to German, maybe will also do Russian afterwards (Russian is actually my first mother tongue, but I use computers in German and would need to look up software-specific terms).

By the way, I wrote a PKGBUILD for the Xfce version of multiload-ng, here it is: https://aur.archlinux.org/packages/xfce4-multiload-ng-plugin-git/ I read that you want to automate the packaging in some way, maybe this helps somehow.

**Comment by udda** [comment]
Yeah I noticed the Facebook comment a few days ago (I assume you are the same person judging by the username :smile:)
It's much more useful than you think, as I currently don't have any experience on Arch. I will integrate your (or similar to yours) PKGBUILD file in a future release.

About the issue, I just pushed a commit that should make you able to finish the translation(s). If you encounter any problems, feel free to open another issue.

Just a little note: very soon (some hours to a couple of days at the lastest) part of the UI will be updated, so part of your translation will be outdated/missing. I suggest to wait until new UI comes into place (if you have enough time of course).

**Comment by Photon89** [comment]
Yeah, that was me. :) Note that this is just a package for the Xfce version (with dependencies selected accordingly), since I am using Xfce myself. Let me know if you need help with PKGBUILDS for the other versions.

Thanks for the warning, how can I get notified when the UI update is there?

**Comment by udda** [comment]
I will comment on this issue, mentioning you, so you'll receive a notification (email).

Thank you again!

**Comment by udda** [comment]
Dear @Photon89 , sorry for the late, it was a hard work (changelog in release notes, tomorrow I think).

New UI is in place. Unless minor modifications, translatable strings should be stable.

You can now start translating, thanks a lot for your help! Let me know if you encounter any problem.

**Comment by Photon89** [comment]
Something went wrong, after the update only two graphs were enabled and I cannot access the settings any more to enable the remaining ones. If I click on "Settings" I only get a small dialog with two buttons "Help" and "Close".

**Comment by udda** [comment]
That's strange. Did you `git pull`ed from existing directory or `git clone`d from scratch?

Only two graphs are visible: this is due to internal change of some settings, so they were reset to their default value. This will happen only once with this update.
Settings does not show: seems like Glade files didn't install. Please try with a new source directory, running `./autogen.sh` and `./configure` again.

If this doesn't work:
1. first of all, please open a new issue, as this is a different problem, so we are able to better track it.
2. try running in terminal *standalone* target (make, install, and type in terminal `multiload-ng`) and see if this prints any error
3. upload your **config.log**

I will try to fix as soon as possible. Thank you again.

**Comment by Photon89** [comment]
Thanks, I found that this is a problem with the packaging and the makefile, will open a new issue.

**Comment by Photon89** [comment]
I sent you the German translation by mail. :)

**Comment by udda** [comment]
I see, thank you!
